doc/building.rst

```diff
@@ -170,23 +176,41 @@ Then, in Terminal:
 
 8. Generate the build files:
 
+.. note::
+
+      By default Nextcloud Desktop will build in a protected directory on macOS,
+      so you need to specify a build location. You can do this every time you build,
+      or you can add it to your save build variables, like so:
+      
    .. code-block:: bash
 
-      % source ~/.nextcloud_build_variables
-      % cd ~/Repositories/desktop/build
-      % cmake .. \
-         -DCMAKE_INSTALL_PREFIX=~/Builds
+        % echo 'export CMAKE_INSTALL_PREFIX=~/Builds' >> ~/.nextcloud_build_variables
+      
+      Replace ``~/Builds`` with a different directory if you'd like the build to end up elsewhere.
+   
+..
 
-   .. note::
+   .. code-block:: bash
 
-      Replace ``~/Builds`` with a different directory if you'd like the build to end up elsewhere.
+      % source ~/.nextcloud_build_variables
+      % cd ~/Repositories/desktop/build
+      % cmake ..
 
 9. Compile and install:
 
    .. code-block:: bash
 
       % make install
 
+macOS Installer (i.e. Deployment) Build
+---------------------------------------
+
+In addition to the above, install Packages:
+
+   .. code-block:: bash
+
+      % brew install packages pkgconfig
+
```

Comment:
@FlexW obviously this needs actual instructions... Do you know what the `make` command is for building a `.pkg` installer?

Reply:
This unfortunate not as easy as just invoking an make command. I have never done this myself by hand so I can't guide you here.

Reply:
Yes, we can revisit the installer build instructions later.